src/main/scala/is/hail/stats/SkatModel.scala

```diff
@@ -23,7 +22,7 @@ class SkatModel(q: Double) {
   
   // gramian is the m x m matrix (G * sqrt(W)).t * P_0 * (G * sqrt(W)) which has the same non-zero eigenvalues
   // as the n x n matrix in the paper P_0^{1/2} * (G * W * G.t) * P_0^{1/2}
-  def computePVal(gramian: DenseMatrix[Double]): SkatStat = {
+  def computePval(q: Double, gramian: DenseMatrix[Double], accuracy: Double = 1e-6, iterations: Int = 10000): (Double, Int) = {
```

Comment:
I prefer to make the default values fields of `SkatModel` and use them anywhere we supply default values.

Reply:
I decided to kill SkatModel as it was an artificial separation, moving the functions into Skat.